docs/cookbook/html-conversion.md

```diff
@@ -0,0 +1,980 @@
+        type: 'area',
+        options: {
+          widgets: {
+            '@apostrophecms/rich-text': {
+              toolbar: [
+                'styles',
+                '|',
+                'bold',
+                'italic',
+                'strike',
+                'link',
+                '|',
+                'bulletList',
+                'orderedList'
+              ],
+              styles: [
+                {
+                  tag: 'p',
+                  label: 'Paragraph (P)'
+                },
+                {
+                  tag: 'h3',
+                  label: 'Heading 3 (H3)'
+                },
+                {
+                  tag: 'h4',
+                  label: 'Heading 4 (H4)'
+                }
+              ]
+            },
+            '@apostrophecms/image': {},
+            '@apostrophecms/video': {}
+          }
+        }
+      }
+    },
+    group: {
+      basics: {
+        label: 'Basics',
+        fields: [ 'title', 'headerImage', 'heading', 'subheading', 'author', 'publicationDate', 'main' ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+modules/blog/index.js
+</template>
+</AposCodeBlock>
+
+The `index.js` file for the `blog-page` will also be quite similar. Again, it will have a header image, heading, and subheading.
+
+<AposCodeBlock>
+
+```js
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    perPage: 5
+  },
+  fields: {
+    add: {
+      headerImage: {
+        type: 'area',
+        label: 'Header image',
+        required: true,
+        options: {
+          widgets: {
+            '@apostrophecms/image': {}
+          },
+          max: 1
+        }
+      },
+      heading: {
+        type: 'string',
+        label: 'Heading'
+      },
+      subheading: {
+        type: 'string',
+        label: 'Subheading'
+      }
+    },
+    group: {
+      basics: {
+        label: 'Basics',
+        fields: [
+          'title',
+          'headerImage',
+          'heading',
+          'subheading'
+        ]
+      }
+    }
+  }
+};
+
+```
+
+<template v-slot:caption>
+modules/blog-page/index.js
+</template>
+</AposCodeBlock>
+
+In the code above we are adding a `perPage` option of `5`. This will limit the number of blog articles shown on the "Home" page. This can be adjusted to best serve your needs.
+
+### The blog index.html page
+
+The "Home" page of the template is essentially an `index.html` page that lists all of the blog articles. Just like with the default page, we are going to add our header to a `beforeMain` block. Within the `main` block, we will copy the `Main Content` section from the `index.html` HTML template. To convert it to dynamically show all of the blog articles from our site we will delete all of the code in each of the `Post preview` sections except the first. Then, we will wrap the first `Post preview` section in a `for` loop. Finally, we will modify the `Pager` section to show both newer and older posts.
```

Comment:
```suggestion
The "Home" page of the template is essentially an `index.html` page that lists all of the blog articles. Just like with the default page, we are going to add our header to a `beforeMain` block. Within the `main` block, we will copy the "Main Content" section from the `index.html` HTML template. To convert it to dynamically show all of the blog articles from our site we will delete all of the code in each of the "Post preview" sections except the first. Then, we will wrap the first "Post preview" section in a `for` loop. Finally, we will modify the "Pager" section to show both newer and older posts.
```

Not sure if those should be formatted as code.

Reply:
I want people to find them in the code, so I added the rest of the comment formatting.